saleor/plugins/webhook/subscription_webhooks/fixtures.py

```diff
@@ -0,0 +1,868 @@
+import pytest
```

Comment:
It seems like the whole `subscription_webhooks` directory should be in `tests` as there is only test-related stuff.

Reply:
It is indeed. Since `subscription_payload.py` was moved there are only test related files here. I'll move them/